src/components/page-replica/index.tsx

```diff
@@ -66,16 +79,19 @@ const PageReplica = ({
       />
       <AsyncResource asyncState={environmentState}>
         {replica && (
-          <Replica
-            logState={pollLogsState}
-            replica={replica}
-            title={
-              <Typography>
-                Replica <strong>{smallReplicaName(replicaName)}</strong>,
-                component <strong>{componentName}</strong>
-              </Typography>
-            }
-          />
+          <>
+            <Replica
+              logState={pollLogsState}
+              replica={replica}
+              downloadOverride={downloadOverride}
+              title={
+                <Typography>
+                  Replica <strong>{smallReplicaName(replicaName)}</strong>,
+                  component <strong>{componentName}</strong>
+                </Typography>
+              }
+            />
+          </>
```

Comment:
Remove unneeded extra fragment tag

```suggestion
          <Replica
            logState={pollLogsState}
            replica={replica}
            downloadOverride={downloadOverride}
            title={
              <Typography>
                Replica <strong>{smallReplicaName(replicaName)}</strong>,
                component <strong>{componentName}</strong>
              </Typography>
            }
          />
```